src/tainting/Dataflow_tainting.ml

```diff
@@ -196,6 +196,10 @@ module Top_sinks = struct
     | Some m -> m.range =*= m'.range
 end
 
+type java_props_cache = (string * G.SId.t, IL.name) Hashtbl.t
+
+let mk_empty_java_props_cache () = Hashtbl.create 30
```

Comment:
at some point, I would love for this to be separated into a different file, or something (with the rest of the java-specific stuff), similarly to what's been done for naming in Pro Engine. `Dataflow_tainting` is getting really hard to read.

Reply:
I plan to do some clean ups!